.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: build
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - Language/**
+      - tests/Language/**
+      - .github/workflows/build.yml
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - Language/**
+      - tests/Language/**
+      - .github/workflows/build.yml
+
+jobs:
+
+  main:
+    name: Build [PHP ${{ matrix.php-versions }}]
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: ${{ matrix.php-versions }}
+          tools: composer
+          extensions: intl, mbstring, xdebug
+          coverage: xdebug
+
+      - name: Validate composer.json
+        run: composer validate --strict
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Create composer cache directory
+        run: mkdir -p ${{ steps.composer-cache.outputs.dir }}
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-
+
+      - name: Setup Composer's GitHub OAuth access to prevent rate limiting
+        run: composer config --global github-oauth.github.com ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@lonnieezell will have to confirm whether we have a `GITHUB_TOKEN` available on this repo.

Reply:
GITHUB_TOKEN is generated automatically on each workflow run, so no need to setup it in Secrets.